vulnerabilities/models.py

```diff
@@ -836,7 +836,7 @@ class Meta:
 
     def save(self, *args, **kwargs):
         checksum = hashlib.md5()
-        for field in (self.summary, self.affected_packages, self.references):
+        for field in (self.summary, self.affected_packages, self.references, self.weaknesses):
```

Comment:
Why not dumping only once?
```
key_data = [self.summary, self.affected_packages, self.references, self.weaknesses]
dumped = json.dumps(key_data, separators=(",", ":")).encode("utf-8")
checksum = hashlib.md5(dumped)
```

Reply:
This will change the checksum IMO, we should do it separately as a migration first and then change the function on models.